generator/generator.go

```diff
@@ -183,11 +183,47 @@ func (gen CodeGenerator) stubbedMethodImplementation(method *ast.Field) *ast.Fun
 		if _, ok := t.(*ast.Ellipsis); ok {
 			ellipsisPos = token.Pos(i + 1)
 		}
+
+		if tArray, ok := t.(*ast.ArrayType); ok && tArray.Len == nil {
+			copyName := name + "Copy"
+			bodyStatements = append(bodyStatements,
+				&ast.AssignStmt{
+					Tok: token.DEFINE,
+					Lhs: []ast.Expr{
+						ast.NewIdent(copyName),
+					},
+					Rhs: []ast.Expr{
+						&ast.CallExpr{
+							Fun: ast.NewIdent("make"),
+							Args: []ast.Expr{
+								t,
+								&ast.CallExpr{
+									Fun:  ast.NewIdent("len"),
+									Args: []ast.Expr{ast.NewIdent(name)},
+								},
+							},
+						},
+					},
+				},
+				&ast.ExprStmt{
+					X: &ast.CallExpr{
+						Fun: ast.NewIdent("copy"),
+						Args: []ast.Expr{
+							ast.NewIdent(copyName),
+							ast.NewIdent(name),
+						},
+					},
+				})
+			paramValuesToRecord = append(paramValuesToRecord, ast.NewIdent(copyName))
+		} else {
+			paramValuesToRecord = append(paramValuesToRecord, ast.NewIdent(name))
+		}
+		paramValuesToPassToStub = append(paramValuesToPassToStub, ast.NewIdent(name))
 	})
 
 	stubFuncCall := &ast.CallExpr{
 		Fun:      stubFunc,
-		Args:     paramValues,
+		Args:     paramValuesToPassToStub,
```

Comment:
I absolutely love this change. These variable names were so generic before, and now it makes sense!